Makefile

```diff
@@ -1,24 +1,20 @@
-CFLAGS += -DSHAPE -DCOLOR -Wall -pedantic -ansi -D_XOPEN_SOURCE
-LDLIBS = -lXext -lX11
-PREFIX ?= /usr
-BIN = $(DESTDIR)$(PREFIX)/bin
+include config.mk
 
-MANDIR = $(DESTDIR)$(PREFIX)/share/man/man1
-MANSUFFIX = 1
+OBJ = 9wm.o event.o manage.o menu.o client.o grab.o cursor.o error.o
 
-all: 9wm
-
-9wm: 9wm.o event.o manage.o menu.o client.o grab.o cursor.o error.o
+9wm: $(OBJ)
+	$(CC) $^ $(LDLIBS) -o $@ 
 
 install: 9wm
 	mkdir -p $(BIN)
-	cp 9wm $(BIN)/9wm
-
-install.man:
+	cp -f $< $(BIN)/$<
```

Comment:
> But I'm going to need a lot of convincing that this is an improvement

The change aims to reduce code in a grammar sense. If desire to install, use `install` target to install 9wm. That process must install related 9wm stuff (e.g, the manual page).

> If you're going to go this route, you should probably have something more like
>
>```Makefile
>install: install.bin install.man
>```

No argues. It can be a great thing.